source/logrepl/snapshot_test.go

```diff
@@ -70,7 +70,7 @@ func TestLifecycle(t *testing.T) {
 			"table":  table,
 		},
 	})
-	is.NoErr(s.Teardown(ctx)) // TODO: Should return an error
+	is.Equal(ErrSnapshotInterrupt.Error(), s.Teardown(ctx).Error())
```

Comment:
We should make sure the error is exactly of the type we expect using `errors.Is`.